### DISCUSSION TITLE
[FeatureRequest] Date formatting: add the ability to format values in UTC

### DISCUSSION THREAD

**Discussion by soulgalore** [grafana/grafana#38187] [Dashboards, Panels & Visualizations]
**What happened**:
Creating a DataLink and converting the time (from epoch UTC) using `${__value.time:date:YYYY-MM-DD-HH-mm}` converts the date using the current browser timezone instead of expressing the time in UTC. The time is converted independently of the timezone you choose in Grafana (and that is correct I think). The problem is that the date is not expressed in UTC.

I create a DataLink to and add the Z to show timezone:
<img width="708" alt="Screenshot 2020-12-10 at 19 29 45" src="https://user-images.githubusercontent.com/540757/101814270-3178f380-3b1e-11eb-89c9-8261c031b530.png">

Looking at the link (the time in Grafana is UTC):
<img width="265" alt="Screenshot 2020-12-10 at 19 30 02" src="https://user-images.githubusercontent.com/540757/101814308-435a9680-3b1e-11eb-9343-6df24cfbdfd7.png">

Then clicking in the link to show the conversion:
<img width="466" alt="Screenshot 2020-12-10 at 19 30 13" src="https://user-images.githubusercontent.com/540757/101814382-5c634780-3b1e-11eb-9345-548b1f01df3e.png">

**What you expected to happen**:
I expect the UTC time for the field to be UTC when converted using date.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version:Grafana v7.3.4 (14c494085e)
- Data source type & version: Graphite  1.1.5
- OS Grafana is installed on: Grafana Docker image
- User OS & Browser: Big Sur, Safari
- Grafana plugins:
- Others:

**Comment by hugohaggmark** [comment]
I took a quick look at this and I think we need to expand the `formatRegistry` with an utc option. Not sure this is a bug, seems to be more like a feature request. https://github.com/grafana/grafana/blob/5d4910dd52c495f315cd29ac61425a5428022c25/public/app/features/templating/formatRegistry.ts#L171

**Comment by soulgalore** [comment]
Hi @hugohaggmark so right now, what time is used, is it the browser time or the one that is chosen in the Grafana GUI? Thinking if I can find a way to work around it.

**Comment by hugohaggmark** [comment]
Hi @soulgalore, looking at the code it seems to be browser time but it could also be "the chosen one", it's hard to tell without digging deeper into the code.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
